src/main/resources/templates/java/test/projectTemplate/pom.xml

```diff
@@ -100,6 +100,15 @@
                         <version>6.28.0</version>
                     </dependency>
                 </dependencies>
+                <configuration>
```

Comment:
I think we now have two `pom.xml` files right? One for eclipse and one for maven. This change should also be done for the other right?